packages/rn-tester/android/app/src/main/res/values/styles.xml

```diff
@@ -1,9 +1,9 @@
 <resources>
-
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
         <!-- Customize your theme here. -->
         <item name="android:textColor">#000000</item>
+        <item name="android:editTextBackground">@drawable/edit_text</item>
```

Comment:
Re: @ShikaSD 
It would be great to add documentation why this file is needed + better prefix for the resource as well

Reply:
>Not sure it is the best way to resolve this issue, as I believe that it should affect other views as well. Also including a random .xml is a nice workaround but I am not sure we want to ship it with the framework

@lunaleaps  Do you mean that I have to prepare an entire new approach to fix this issue (https://github.com/facebook/react-native/pull/29452#issuecomment-912257356 and https://github.com/facebook/react-native/pull/29452#issuecomment-915687725)? Could ShikaSD give me some help or feedback on how to solve this issue? https://github.com/facebook/react-native/pull/29452#issuecomment-915687725. Thanks

Reply:
Let's have a discussion about @ShikaSD's concerns -- he is currently away and will return next week.

Reply:
Hey, thanks for approaching this, do you know if this repros everywhere or only on particular Android version?

My comment was mostly about the fact that setting XML through themes will only affect the edit text with this theme applied. I don't think we can guarantee that every React Native app uses the default theme or the material background, so here the proposed solution feels like a workaround that could help only in some cases. 🙂 
I also noticed the reports in the original issue thread with slightly different traces/views, meaning that this patch might not fix the issue completely, as we cannot control which drawables people use in their applications.

I guess the proper solution would be to patch Android SDK to actually fix this issue, but as long as this is not possible, maybe we can find a more stable workaround that can at least catch the issue before the crash 😅

Reply:
I also checked the comments in the thread, I am not sure what is the proposed solution in both https://github.com/facebook/react-native/pull/29452#issuecomment-915687725 or https://github.com/facebook/react-native/pull/29452#issuecomment-912257356 is. The Drawables mentioned in both comments assume that we have some access to definition of those, but it seems like the original problem comes from XML defined drawables in design/appcompat libs which we cannot really influence.

I think we should be good to go with the temporary fix for this particular edit text with minor edits (@lunaleaps ported some comments above about adding some reference/better naming), but still concerned if the drawables are also crashing on other views.